5/slim/Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM mhart/alpine-node:5.5.0
+FROM mhart/alpine-node:5
 
 # Add gcc and git support for native dependencies.
-RUN apk add --update g++ gcc git make python && rm -rf /var/cache/apk/*
+RUN apk --no-cache --virtual buildDependencies add g++ gcc git make python
```

Comment:
Hmm, this made me wonder if we really need python support? Also, should we upgrade to python3 here?